CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.2.1] - 2019-03-01
+
+### Fixed
+
+- `ZeroDivisionError` on systems with limited precision time.
+
+## [0.2.0] - 2019-02-26
+
+### Added
+
+- Allow the proportional term to be monitored properly through the components-property when _proportional on measurement_ is enabled.
+
+### Fixed
+
+- Bump in output when re-enabling _auto mode_ after running in _manual mode_.
+
+## [0.1.5] - 2010-01-31
+
+### Added
+
+- The ability to see the contributions of the separate terms in the PID
```

Comment:
**Note**: Beyond this addition, I have the feeling that [Divide D term by delta time](https://github.com/m-lundberg/simple-pid/commit/af2ab1b022781386e8e227df3f85d6d9bf6ff9ef) may have been a bug fix, but I am not yet familiar enough with the domain to tell for sure.

_The fact that you tagged this release as v0.1.5 coming from v0.1.4 also made me expect to find a bug fix, but I might be misinterpreting your intention._ : )

Reply:
You are correct, that was a bug fix. Something like this could be added:

```suggestion
- The ability to see the contributions of the separate terms in the PID

### Fixed

- D term not being divided by delta time, leading to wrong output values
```